spec/features/admin/poll/questions/answers/images/images_spec.rb

```diff
@@ -7,8 +7,57 @@
     login_as(admin.user)
   end
 
-  pending "Index"
-  pending "Create"
-  pending "Destroy"
+  context 'Index' do
+    scenario 'Answer with no images' do
+      answer = create(:poll_question_answer,
+                      question: create(:poll_question))
 
-end
\ No newline at end of file
+      visit admin_answer_images_path(answer)
+
+      expect(page).to_not have_css("img[title='']")
+    end
+
+    scenario 'Answer with images' do
+      answer = create(:poll_question_answer,
+                      question: create(:poll_question))
+      image = create(:image, imageable: answer)
+
+      visit admin_answer_images_path(answer)
+
+      expect(page).to have_css("img[title='#{image.title}']")
+      expect(page).to have_content(image.title)
+    end
+  end
+
+  scenario 'Add image to answer', :js do
+    answer = create(:poll_question_answer,
+                    question: create(:poll_question))
+    image = create(:image, imageable: answer)
```

Comment:
If we create the image through code instead of through interface, I'm not sure this tests is actually verifying the user can add an image to an answer